nearcore/src/lib.rs

```diff
@@ -221,6 +222,12 @@ pub fn start_with_config_and_synchronization(
         None
     };
 
+    let trie_metrics_arbiter = spawn_trie_metrics_loop(
```

Comment:
Let me add a timer and log the time

Reply:
@andrei-near It typically takes between 8 and 10ms for all shards combined, on my personal node.

Reply:
Just worth keeping in mind that it takes ~10ms when there is 1-2 postponed receipts, it will likely scale up with the number of trie nodes to iterate.

Reply:
I'm curious how frequently does prometheus query the data. There isn't much value in pushing stats more often. If we do require more granularity than prometheus offers I suppose we could use some pre-aggregation on neard side. I think I've seen some smarter gauges that do that automatically.

Reply:
I would guess the Prometheus collection would be 10-30 seconds, based on my experience interacting with Grafana. I'll try to find the exact number.

I agree that 10s frequency would be more than enough in this case.